source/install/ config-ssl-http2-apache2.rst

```diff
@@ -0,0 +1,60 @@
+.. _config-ssl-http2-apache2:
+
+Configuring Apache2 with SSL and HTTP/2 (Unofficial) 
+=====================================
+
+Once you've configured Apache2 as a proxy for your Mattermost Server, the easiest way to enable SSL on Apache2 is via Let's Encrypt and [Certbot](https://certbot.eff.org/#ubuntuxenial-apache). Once installed, simply run:
+``$ certbot --apache``
+And follow the straight forward guide. Afterwards you should find a new configuration file in ``/etc/apache2/sites-available`` which should follow the format: ``mysubdomain.mydomain.com-le-ssl.conf``.
+
+Once opened it should look something like the following:
+
+.. code-block:: none
+	<IfModule mod_ssl.c>
+	<VirtualHost *:443>
+		ServerName mysubdomain.mydomain.com
+		ServerAdmin hostmaster@mydomain.com
+		ProxyPreserveHost On
+
+		# setup the proxy
+		<Proxy *>
+		  Order allow,deny
+		  Allow from all
+		</Proxy>
+
+		RewriteEngine On
+		RewriteCond %{REQUEST_URI} ^/api/v3/users/websocket [NC,OR]
+		RewriteCond %{HTTP:UPGRADE} ^WebSocket$ [NC,OR]
+		RewriteCond %{HTTP:CONNECTION} ^Upgrade$ [NC]
+		RewriteRule .* ws://127.0.0.1:8065%{REQUEST_URI} [P,QSA,L]
+		RewriteCond %{DOCUMENT_ROOT}/%{REQUEST_FILENAME} !-f
+		RewriteRule .* http://127.0.0.1:8065%{REQUEST_URI} [P,QSA,L]
+
+		<Location /api/v3/users/websocket>
```

Comment:
@JeffSchering is v3.9 of MM using api/v4 for sockets? My set up seems to be connecting to with the v3 api.